### DISCUSSION TITLE
How can I in spatie/laravel-medialibrary to keep info about youtube key?

### DISCUSSION THREAD

**Discussion by sergeynilov** [spatie/laravel-medialibrary#3322] [Q&A]
In laravel 8 app using spatie/laravel-medialibrary 9 for storing of images and videos in adminarea I also use youtube keys 
(only key, not url) which can be attached for some item. If there is a way to store youtube key im media table of laravel-medialibrary
and have one common model both for media files uploaded by user and by youtube key ?

**Comment by francoism90** [comment]
You mean the YouTube ID?

One can use custom properties: https://spatie.be/docs/laravel-medialibrary/v10/advanced-usage/using-custom-properties

**Comment by sergeynilov** [comment]
I know howto use setCustomProperty, but I need to create a media file row, not just additive property to existing media.
